data-structures/linked_list/mergeList.py

```diff
@@ -0,0 +1,33 @@
+# from linked_list import LinkedList
+
+
+def ll_merge(arg2, arg1):
+    if arg2._len <= 0 and arg1._len <= 0:
+        return False
+
+    if arg2.head is None:
+        return arg1
+    if arg1.head is None:
+        return arg2
+
+    if True:
```

Comment:
This is an entirely pointless `if` statement, given this condition. The `else` statement will never execute, so either change this condition or remove the conditions all together.